presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -290,18 +332,33 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
         return builder.build();
     }
 
-    private TupleDomain<ColumnHandle> pruneColumns(TupleDomain<ColumnHandle> predicate, Set<TpchColumn> columns)
```

Comment:
You are modifying code that you added two commits earlier. Can you squash those commits?